packages/elements/src/components/ino-popover/ino-popover.tsx

```diff
@@ -242,8 +249,10 @@ export class Popover implements ComponentInterface {
 
     const options: Partial<Props> = {
       allowHTML: true,
+      theme: this.colorScheme,
```

Comment:
Nice! IMO it's the better approach to use the documented way to adjust styling of tippy instances. Currently the ino-tooltip uses an custom approach to add stylings. It would be nice to be consistent  in codebase on handling Tippy instances. It would be great to refactor ino-tooltip  to inject stylings via theming as well.

Reply:
Do you mean this approach [TippyJS: creating-a-theme](https://atomiks.github.io/tippyjs/v6/themes/#creating-a-theme)? Because I used this approach to create theme-stylings which resemble our designs in [_tooltip-base.scss line 41 to 56](packages/elements/src/components/base/_tooltip-base.scss). 

`this.colorScheme` returns the name of this themes (`light`, `dark` or `primary`) and applies it to the tippy Instance. 

`ino-tooltip` uses the same theme styling from this file, see [PR #975](https://github.com/inovex/elements/pull/975)